web/static/lib/jquery/jquery.dirty.js

```diff
@@ -0,0 +1,308 @@
+/*
+ * Dirty
+ * jquery plugin to detect when a form is modified
+ * (c) 2016 Simon Taite - https://github.com/simon-reynolds/jquery.dirty
+ * originally based on jquery.dirrty by Ruben Torres - https://github.com/rubentd/dirrty
+ * Released under the MIT license
+ */
+
+(function($) {
+
+    //Save dirty instances
+    var singleDs = [];
+    var dirty = "dirty";
+    var clean = "clean";
+    var dataInitialValue = "dirtyInitialValue";
+    var dataIsDirty = "isDirty";
+
```

Comment:
Is there any reason why you are using `var` instead of  `let` or `const`? If not, then I think it is preferable to avoid using `var` 

(see https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var)

Reply:
I absolutely agree, but this is a library I downloaded (see the comment at the top of the file), and I generally don't think we should put resources into cleaning up those 😅